### PR TITLE
update lock file for 5.5

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: Gemfile.jruby-1.9.lock.release [hunk 1]
```diff
@@ -258,7 +240,7 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       nokogiri
       xml-simple
-    logstash-input-beats (3.1.15-java)
+    logstash-input-beats (4.0.1-java)
```
Comment (jakelandis) [inline Gemfile.jruby-1.9.lock.release]: This should be 3.x (likely 3.1.17) not 4.x

Comment (suyograo) [inline Gemfile.jruby-1.9.lock.release]: good catch @jakelandis

File: Gemfile.jruby-1.9.lock.release [hunk 2]
```diff
@@ -333,7 +318,7 @@ GEM
       sequel
       tzinfo
       tzinfo-data
-    logstash-input-kafka (6.3.0)
+    logstash-input-kafka (5.1.7)
```
Comment (suyograo) [inline Gemfile.jruby-1.9.lock.release]: I had accidentally pushed major versions

**Comment by ph** [comment]
good catch @jakelandis for beats!

**Comment by suyograo** [comment]
Waiting for CI

**Pull request merged**